app/util.js

```diff
@@ -109,12 +109,19 @@ const searchword = location => {
             const q = location.replace(`${word} `, "")
             if (q && location.startsWith(`${word} `)) {
                 const queries = q.split(",").map(item => item.trim())
-                let i = 0
                 let urlString = url
-                while (urlString.includes("%s")) {
-                    urlString = urlString.replace(/%s/, encodeURIComponent(queries[i]))
-                    i += 1
+                let counter = 1
+                const patternMatches = (urlString.match(/%s/g) || []).length
+                while (urlString.includes("%s") && counter < patternMatches) {
+                    if (0 in queries) {
```

Comment:
Is this the same as `if (queries[0]) {` if so, I would prefer that syntax for consistency with the rest of Vieb's code.